.github/workflows/build_images_uv.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
         with:
           platforms: linux/amd64,linux/arm64
           push: true
+          target: "main_image"
           tags: |
             mzdotai/lumigator:backend_dev_${{ env.REPOVERSION }}
```

Comment:
Maybe factoring `mzdotai/lumigator` as a variable? So that things are kept consistent if the registry changes.

Reply:
Hey @javiermtorres yep, it's a good idea, but I will introduce it in another PR with the semver that you mentioned in your other comment 👍

Reply:
Ok, np!!!!